src/viur/core/bones/key.py

```diff
@@ -47,7 +47,10 @@ def singleValueFromClient(self, value, skel, bone_name, client_data):
                 return self.getEmptyValue(), [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, e.args[0])]
         else:
             try:
-                key = db.normalizeKey(db.Key.from_legacy_urlsafe(value))
+                if isinstance(value, str):
+                    key = db.normalizeKey(db.Key.from_legacy_urlsafe(value))
+                elif isinstance(value, db.Key):
+                    key = db.normalizeKey(value)
             except:
```

Comment:
```suggestion
            except Exception as exc:
                logging.exception(f"Failed to normalize {value}: {exc}")
```

Reply:
@ArneGudermann can you please add this proposal to your PR in order for merging?